nipype/workflows/fmri/fsl/preprocess.py

```diff
@@ -865,10 +896,24 @@ def create_susan_smooth(name="susan_smooth", separate_masks=True):
     """
     Define a function to get the brightness threshold for SUSAN
     """
-    susan_smooth.connect(inputnode, 'fwhm', smooth, 'fwhm')
-    susan_smooth.connect(inputnode, 'in_files', smooth, 'in_file')
-    susan_smooth.connect(median, ('out_stat', getbtthresh), smooth, 'brightness_threshold')
-    susan_smooth.connect(merge, ('out', getusans), smooth, 'usans')
+    # if you are going to iterate over multiple values of fwhm
+    if list_fwhms:
+        susan_smooth.connect([
+            (inputnode, multi_inputs, [('in_files', 'in_files'),
+                                       ('fwhm', 'fwhms'),
+                                       ('mask_file', 'mask_files')]),
+        ])
+        susan_smooth.connect(median, ('out_stat', getbtthresh), multi_inputs, 'median_out')
+        susan_smooth.connect(merge, ('out', getusans), multi_inputs, 'merge_out')
+        susan_smooth.connect(multi_inputs, 'multi_fwhms', smooth, 'fwhm')
+        susan_smooth.connect(multi_inputs, 'multi_in_files', smooth, 'in_file')
+        susan_smooth.connect(multi_inputs, 'multi_median_out', smooth, 'brightness_threshold')
+        susan_smooth.connect(multi_inputs, 'multi_merge_out', smooth, 'usans')
+    else:
+        susan_smooth.connect(inputnode, 'in_files', smooth, 'in_file')
+        susan_smooth.connect(inputnode, 'fwhm', smooth, 'fwhm')
+        susan_smooth.connect(median, ('out_stat', getbtthresh), smooth, 'brightness_threshold')
+        susan_smooth.connect(merge, ('out', getusans), smooth, 'usans')
```

Comment:
I think this will be more readable with a multi-connect statement:

```Python
susan_smooth.connect([
    (inputnode, multi_inputs, [('in_files', 'in_files'),
                               ('fwhm', 'fwhms')]),
    (median, multi_inputs, [(('out_stat', getbtthresh), 'btthresh')]),
    (merge, multi_inputs, [(('out', getusans), 'usans')]),
    (multi_inputs, smooth, [('in_file', 'in_file'),
                            ('fwhm', 'fwhm'),
                            ('btthresh', 'brightness_threshold'),
                            ('usans', 'usans')]),
    ])
```

Reply:
I think so too, but I didn't want to change the way they had it in case that was their coding standard, but with your recommendation I'll change it.

Reply:
My feeling is this workflow was written to be readable, as an example for writing them. I think having a bit of a mix where we choose the most readable at any given point is within the spirit of the thing.